manifests/service.pp

```diff
@@ -0,0 +1,29 @@
+# PRIVATE CLASS: do not call directly
```

Comment:
To make this a private class that is not called directly, it should not inherit from params. Also it should not have its own spec test, since it would be part of using the class `mrepo`. You would go back to using `$service_enable = $mrepo::service_enable`, etc., remove `inherits ::mrepo::params`, and insert the appropriate tests inside the file `spec/classes/mrepo_spec.rb`. In the test, change the parameters given to the `mrepo` class as necessary to generate the "downstream" results here in `mrepo::service` and the resulting catalog contents.

Alternatively, you can make this a public class. I do think that keeping it private is the more correct solution, even if it's a bit more complex, though I'm not strongly attached to either method. If you want some more assistance on this, you can find me on irc or slack or ask any other vox pupuli member for assistance. Thanks!